ui/app/pages/token/index.scss

```diff
@@ -1,3 +1,152 @@
 @import 'fee-card/index';
 @import 'exchange-rate-display/index';
 @import 'actionable-message/index';
+
+.token {
+  &__search-token {
+    margin: 24px;
+    border: 1px solid #fff;
+    box-sizing: border-box;
+    box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+    border-radius: 6px;
+    min-height: 297px;
+    width: 100%;
+
+    .searchable-item-list {
```

Comment:
I don't think this is a good idea. The class `.searchable-item-list` is an internal implementation detail of the `SearchableItemList` component. Using the class directly outside of that component dramatically complicates refactoring.

Additional styles should be passed in via an optional custom classname.